src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1238,7 +1238,7 @@ private void LoadTransitions()
                 }
                 if (trans)
                 {
-                    transDictionary.Add(trans.Name, trans);
+                    transDictionary?.Add(trans.Name, trans);
```

Comment:
> Since `transDictionary` is initialized at declaration, i think null-checking is unnecessary. instead, it is better to make `transDictionary` field readonly to prevent null assignment.

OK. I will revert it